### PR TITLE
Add tool to send events

### PR DESCRIPTION
**Pull request by c-w**
This pull request adds a tool to feed the agogosml input sources (Kafka and EventHub). This enables manual kick-start of the pipeline, redriving training data, etc.

The tool can be used like so:

```bash
python3 -m agogosml.tools.sender
```

### All Submissions:

- [x] Have you followed the guidelines in our [Contributing](https://github.com/Microsoft/agogosml/blob/master/CONTRIBUTING.rst) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Microsoft/agogosml/pulls) for the same update/change?
- [x] Does your PR follow our [Code of Conduct](https://opensource.microsoft.com/codeofconduct/)?

- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] Does each method or function "do one thing well"? Reviewers may recommend methods be split up for maintainability and testability.
- [x] Is this code designed to be testable? 
- [x] Is the code documented well?
- [x] Does your submission pass existing tests (or update existing tests with documentation regarding the change)?
- [x] Have you added tests to cover your changes?
- [x] Have you linted your code prior to submission?
- [x] Have you updated the documentation and README?
- [x] Is PII treated correctly? In particular, make sure the code is not logging objects or strings that might contain PII (e.g. request headers).  **Not applicable: no PII included**
- [x] Have secrets been stripped before committing? **Not applicable: no secrets included**

### CODE REVIEW

File: agogosml/agogosml/tools/sender.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+"""Script to push inputs to an agogosml client."""
+from argparse import ArgumentParser
+from argparse import ArgumentTypeError
+from argparse import FileType
```
Comment (sayar) [inline agogosml/agogosml/tools/sender.py]: Nit: Any reason why we shouldn't collapse these imports into one statement?

File: agogosml/agogosml/tools/sender.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+from inspect import getsourcefile
+from pathlib import Path
+from pkgutil import walk_packages
+from sys import stdin
+from typing import Dict
+from typing import IO
+from typing import Iterable
+from typing import Tuple
+from typing import Type
+from typing import TypeVar
```
Comment (sayar) [inline agogosml/agogosml/tools/sender.py]: Nit: Same with these import statements.

File: agogosml/setup.cfg [hunk 3]
```diff
@@ -37,4 +37,5 @@ norecursedirs =
 	.env
 	.venv
 collect_ignore = ['setup.py']
-
+addopts = --doctest-modules
```
Comment (sayar) [inline agogosml/setup.cfg]: 👍

File: agogosml/agogosml/tools/sender.py (hunk 1 above)
Comment (c-w) [reply agogosml/agogosml/tools/sender.py]: Makes for cleaner diffs. Strongly recommended IMO. I'll add the isort linter later to enforce consistent import styles.

File: agogosml/agogosml/tools/sender.py (hunk 2 above)
Comment (c-w) [reply agogosml/agogosml/tools/sender.py]: See reply above.

File: agogosml/agogosml/tools/sender.py (hunk 1 above)
Comment (c-w) [reply agogosml/agogosml/tools/sender.py]: See PR https://github.com/Microsoft/agogosml/pull/217.

**Pull request merged**